File: Talleres-1B/Taller-6/numPrimos.py
#Inicialización de variables
divisor = 0; i = 0; j = 0

#Proceso y Salida
n = int(input("¿Cuántos números va a ingresar?: "))

for i in range(1, n+1):
    num = int(input("Ingrese un número: "))

    #Verificar si un número es par o impar
    if num % 2 == 0:
        print("El número ",num," es PAR \n")
    else:
        print("El número ",num," es IMPAR \n")
    
    #Verificar si un número es primo
    j = 1
    while j<=num:
        if num % j == 0:
            divisor += 1
        
        j += 1
    if divisor == 2:
        print("El número ",num," es Primo \n")
    else:
        print("El número ",num," no es Primo \n")
    divisor = 0

File: Talleres-1B/Taller-6/CRepetitivos/src/crepetitivos/numPrimos.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package crepetitivos;
import java.util.Scanner;
/**
 *
 * @author SALAA
 */
public class numPrimos {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        
        //Declaración e Inicialización de las Variables
        int i, n, num;
        int j, divisor;
        
        n=0; num=0; divisor=0;
        //Ingreso de Datos
        System.out.println("Ingrese el límite: ");
        n = leer.nextInt();
        
        for (i=1; i<=n; i++){
            System.out.println("Ingrese un número: ");
            num = leer.nextInt();
            
            //Verificar si un número es par o impar
            if (num % 2 ==0){
                System.out.println("El número " +num +" es PAR \n");
            }
            else{
                System.out.println("El número " +num +" es IMPAR \n");
            }
            
            //Verificar si un número es primo
            j = 1;
            while (j<=num){
                if (num % j==0){
                    divisor = divisor + 1;
                }
                j = j+1;
            }
            if (divisor ==2){
                System.out.println("El número " +num +" es Primo \n");
            }
            else {
                System.out.println("El número " +num +" no es Primo \n");
            }
            divisor = 0;
        }
    }
}

